openquake/hazardlib/gsim/kotha_2020.py

```diff
@@ -162,6 +193,30 @@ def get_magnitude_scaling(C, mag):
                     C["e1"] + C["b3"] * d_m)
 
 
+def get_dl2l(ctx, imt, delta_l2l_epsilon):
+    """
+    Returns rupture source specific delta_l2l values. The method
+    retrieves the delta_l2l and standard error of delta_l2l values.
+    if delta_l2l_epsilon is provided, standard error of delta_c3 
+    will be included. A delta_l2l_epsilon value of +/- 1.6 gives 
+    the 95% confidence interval for delta_l2l.
+    """
+    f = []
+    [f.append(Point(lon, lat)) for lon, lat in zip(ctx.hypo_lon, ctx.hypo_lat)]
+    dl2l = np.zeros((len(ctx.hypo_lon), 2), dtype = float)
+    fname = os.path.join(DATA_FOLDER, 'kotha_tectonic_regions.geojson')
```

Comment:
Same suggestion provided for the path term

Reply:
incorporated.